LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Hevar Hoshang Ahmed
+
```

Comment:
- [Optionl] Hello team, Good job making the changes, but while you do so, please remember this is a group project, and as such the License file should be granted by the name of both collaborators. For the time being I can see only `Hevar Hoshang Ahmed`.